docs/dictionary/command/encrypt-using-rsa.lcdoc

```diff
@@ -38,22 +35,22 @@ Description:
 Use the <encrypt using rsa> command to encrypt a message using RSA
 public key encryption.
 
-Use the form 
-encrypt <message> with public key <key> 
-to encode a message that you only want to be decoded by the holder of
-the private key.
-
-Use the form 
-encrypt <message> with private key <key> 
-to encode a message that a receiver can then verify has come from one of
-the holders of the private key (this is a signing operation).
-
-Generating key pairs
-Public-private key pairs can be generated using the OpenSSL suite of
-command-line tools. For example: openssl genrsa -out private_key.pem 512
-openssl rsa -pubout -in private_key.pem -out public_key.pem
-Will generate a key pair of size 512-bits, placing the private key in
-private_key.pem and the public key in public_key.pem.
+To encode a message that you only want to be decoded by the holder of
+the private key, use the form:
+
+    encrypt <message> with public key <key> 
+
+As a signing operation, to encode a message that a receiver can then verify 
+has come from one of the holders of the private key, use the form:
+
+    encrypt <message> with private key <key> 
+
+> **Generating key pairs - **
+> Public-private key pairs can be generated using the OpenSSL suite of
+> command-line tools. For example: openssl genrsa -out private_key.pem 512
+> openssl rsa -pubout -in private_key.pem -out public_key.pem
+> Will generate a key pair of size 512-bits, placing the private key in
+> private_key.pem and the public key in public_key.pem.
 
```

Comment:
@seaniepie We're generally sticking to just the Imporant, Note, Warning and Tip structure for blockquoted sections - there's an option here to make this a `>*Tip:*` I guess, but if you want to keep the heading I suggest doing a markdown section header like `#### Generating key pairs` and then formatting the rest of the paragraph normally. We do this elsewhere (eg the repeat docs).